descriptors/ModuleDescriptor-template.json

```diff
@@ -56,6 +56,16 @@
           "pathPattern": "/query/{query-id}",
           "permissionsRequired": ["fqm.query.async.results.get"]
         },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/query/{query-id}/sortedIds",
```

Comment:
New API required by mod-lists to get sorted list of ids from mod-fqm-manager during list refresh.

Reply:
(no changes or anything here... This just seems like a convenient place to float an idea)

I wonder how resource-intensive it'd be to move the sorting to mod-lists... Then, we'd be able to just request the data for a list by doing a query for a list of fields where `id in (id1, id2, id3, etc)` - on the one hand, it'd simplify the FQM by not needing this endpoint. It'd also relieve some strain on the DB, which is always good. On the other hand, we'd need to make sure each mod-lists instance has enough RAM to sort large lists, which could get expensive.

Another idea: add something to the query request schema to define the sort order. In that case, we could using the existing endpoints to do a query from mod-lists to FQM for a list of fields where `id in (id1, id2, id3, etc)` ordered by a, b, c. Then, we'd still be able to sort in the DB and keep the mod-lists instance size down